src/document-header/index.jsx

```diff
@@ -30,19 +30,22 @@ export default function DocumentHeader({ title, subtitle, backLink, children, de
   function Headings({ title, subtitle }) {
     if (isSticky) {
       return (
-        <h2>
-          {
-            title
-              ? <Fragment>{title}: {subtitle}</Fragment>
-              : subtitle
-          }
-        </h2>
+        <Fragment>
+          <h2>
+            {
+              title
+                ? <Fragment>{title}: {subtitle}</Fragment>
+                : subtitle
+            }
+            {status}
+          </h2>
+        </Fragment>
       );
     }
 
     return (
       <Fragment>
-        { title && <h1>{title}</h1> }
+        { title && <h1>{title} {status}</h1> }
```

Comment:
This will add an additional space to the end of a title when no status is there, would this cause any issues?